front/src/applications/stdcm/hooks/useStdcmResults.ts

```diff
@@ -59,12 +59,12 @@ const useStdcmResults = (
     const getPathProperties = async (_infraId: number, path: PathfindingResultSuccess) => {
       const pathPropertiesParams: PostInfraByInfraIdPathPropertiesApiArg = {
         infraId: _infraId,
-        props: ['electrifications', 'geometry', 'operational_points'],
+        props: ['electrifications', 'geometry', 'operational_points', 'zones'],
         pathPropertiesInput: {
           track_section_ranges: path.track_section_ranges,
         },
       };
-      const { geometry, operational_points, electrifications } =
+      const { geometry, operational_points, electrifications, zones } =
         await postPathProperties(pathPropertiesParams).unwrap();
 
       if (geometry && operational_points && electrifications) {
```

Comment:
Since we know `zones` will be defined and we updated the `pathProperties` type.
We can also remove `electrifications` (check the comment in operationamStudies/types.ts)

```suggestion
      if (geometry && operational_points && zones) {
```